CHANGELOG.md

```diff
@@ -2,13 +2,26 @@
 
 ## main (unreleased)
 
+## 2.0.0 (2022-03-11)
+
 * [#58](https://github.com/Shopify/deprecation_toolkit/pull/58): Drop support for Ruby < 2.6 & Active Support < 5.2. (@sambostock)
 * [#58](https://github.com/Shopify/deprecation_toolkit/pull/58): Ensure compatibility with Rails 7. (@sambostock)
 
+## 1.5.1 (2020-04-28)
+
+* [#46](https://github.com/Shopify/deprecation_toolkit/pull/46): Handle another two part Ruby 2.7 keyword argument deprecation warning. (@casperisfine)
+
+## 1.5.0 (2020-04-14)
+
+* [#42](https://github.com/Shopify/deprecation_toolkit/pull/42): Fix Minitest plugin kicking in when it shouldn't. (@Edouard-chin)
+* [#45](https://github.com/Shopify/deprecation_toolkit/pull/45): Handle two part Ruby 2.7 keyword argument deprecation warning. (@casperisfine)
```

Comment:
On top of the changelog entries, these were also missing releases, which I've since created.